src/main/resources/_config.properties

```diff
@@ -3,7 +3,8 @@
 #=====================================================#
 
 #=================== Global configuration ============#
-url=http://www.gsmarena.com
+url=http://www.gsmarena.com 
+# use it for perfomance test url=https://login.yahoo.com/config/login?.src=fpctx&.intl=us&.lang=en-US&.done=https%3A%2F%2Fwww.yahoo.com
```

Comment:
remove this comment. Your test must work automatically without any manual preliminary fix. Take a look to above comment howto reuse absolute url for page opening